docs/src/recipes/aggregate_index.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c74f7111-1b6c-4454-9770-3f67eeadaca6",
+   "metadata": {},
+   "source": [
+    "# Aggregate events from different indexes\n",
+    "\n",
+    "This recipe applies when you have events indexed by one or more features, and you want to drop some of the index levels and unify the events with the same timestamps.\n",
+    "\n",
+    "For example, assume we have multiple stores with multiple products being sold, so we have daily sale events indexed by store ID and product ID. We want to aggregate sales per store ID, adding up all sales from different product IDs."
```

Comment:
In this example, we aggregate individual sale transactions (such as the timestamp and value of each product purchase) into daily revenue (i.e., the total sales for each day) for each individual store.